packages/nodes-base/credentials/HubspotAppToken.credentials.ts

```diff
@@ -1,4 +1,9 @@
-import { IAuthenticateGeneric, ICredentialTestRequest, ICredentialType, INodeProperties } from 'n8n-workflow';
+import {
```

Comment:
this could be 
```suggestion
import type {
```

Reply:
This happens everywhere in nodes-base. Let's tackle separately?

Reply:
in that case, please feel free to merge.

Reply:
Tracking [here](https://linear.app/n8n/issue/N8N-5688/use-type-imports-in-nodes-base).